Bottom: f2f65066537bd6b23722a1192711e6e6896d9b70
Top:    a84caa62b552a4a64725301dfc07735a34954d25
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 13:54:15 -0500

Refresh of fixup-add-buildbot-ntel-worker

---
diff --git a/ansible-playbook/buildbot.yml b/ansible-playbook/buildbot.yml
index bdad791f..2a5cd968 100644
--- a/ansible-playbook/buildbot.yml
+++ b/ansible-playbook/buildbot.yml
@@ -22,7 +22,7 @@
       new_user_pass: "{{ lookup('password', '/dev/null') }}"
 
 - name: Setup Buildbot worker system
-  hosts: buildbot-worker
+  hosts: builder
   become: true
   become_user: root
   become_method: sudo
